app/views/user/settings.ctp

```diff
@@ -36,282 +36,318 @@ $this->set('title_for_layout', $pages->formatTitle(__('Settings', true)));
 </div>
 
 <div id="main_content">
-    <div class="module options">
+    <div class="options">
         <?php echo $form->create(null, array('action' => 'save_settings')); ?>
+        <div md-whiteframe="1" class="settings-form">
+            <h2><?php __('Options'); ?></h2>
+            <fieldset>
+            <md-list flex role="list" class="flex" >
+                <md-list-item>
+                    <div>
```

Comment:
This div can be removed. Same for all the div's inside the other `<md-list-item>`.